src/NzbDrone.Api/Profiles/LanguageValidator.cs

```diff
@@ -0,0 +1,45 @@
+﻿
+using System.Collections.Generic;
+using System.Linq;
+using FluentValidation;
+using FluentValidation.Validators;
+using NzbDrone.Api.Profiles;
+
+namespace NzbDrone.Core.Validation
+{
+    public static class LanguageValidation
+    {
+        public static IRuleBuilderOptions<T, IList<ProfileLanguageItemResource>> MustHaveAllowedLanguage<T>(this IRuleBuilder<T, IList<ProfileLanguageItemResource>> ruleBuilder)
```

Comment:
It is pretty much the same that with quality. Maybe change both to MustHaveAnAllowed<Quality/Language>?